lua/care/menu/init.lua

```diff
@@ -233,20 +233,15 @@ end
 
 function Menu:select_next(count)
     count = count or 1
-    self.index = self.index + count
-    if self.index > #self.entries then
-        self.index = self.index - #self.entries - 1
-    end
+    self.index = (self.index + count) % #self.entries
```

Comment:
just tested this locally and it doesn't work
can't select the last entry in the menu